### DISCUSSION TITLE
IPv4 address with CIDR mask

### DISCUSSION THREAD

**Discussion by viesturs-veckalns** [Security-Onion-Solutions/securityonion#12077] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

13

### Storage for /

100

### Storage for /nsm

idk

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Installing SO in Virtualbox hosted in Windows.
What answer to provide to this question:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/134596935/1c4895a2-9778-42e8-aafc-e78847325422)
?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
This is where you would assign the IP address to the Security Onion installation. This IP address will be used on the management interface which would most likely be configured as either NAT or Bridged in VirtualBox. This is how the Security Onion VM will connect to the Internet and how you will connect to the web interface of Security Onion from your Windows host or other machines.

For more information, please see:
https://docs.securityonion.net/en/2.4/virtualbox.html#creating-vm
